tests/unit/vqb.spec.ts

```diff
@@ -16,11 +14,39 @@ describe('Vqb', () => {
     expect(store.state.isEditingStep).toBeFalsy();
   });
 
+  it('should instantiate a ForRenameStep component', () => {
+    const store = setupStore({ isEditingStep: true });
+    const wrapper = shallowMount(Vqb, {
+      store,
+      localVue,
+      data: () => {
+        return { formToInstantiate: 'FormRenameStep' };
+      },
+    });
+    const form = wrapper.find('formrenamestep-stub');
```

Comment:
to be more consistent and precise, we have to call it a wrapper
```suggestion
    const formWrapper = wrapper.find('formrenamestep-stub');
```